patch/llpcPatchResourceCollect.cpp

```diff
@@ -218,22 +218,6 @@ void PatchResourceCollect::visitCallInst(
             m_deadCalls.insert(&callInst);
         }
     }
-    else if (mangledName.startswith(LlpcName::ImageCallPrefix))
```

Comment:
where is the implementation for removing deal image operations? could you help to point it out? thanks!

Reply:
Any dead code gets removed in the standard LLVM dead code removal pass.

Reply:
got it. thanks! please double check whether the dead image operations are removed before entry point mutate, since it will affect the user data layout.